archaius2-test/src/main/java/com/netflix/archaius/test/TestCompositeConfig.java

```diff
@@ -0,0 +1,62 @@
+package com.netflix.archaius.test;
+
+import java.util.Iterator;
+import java.util.Properties;
+
+import com.netflix.archaius.api.Config;
+import com.netflix.archaius.api.config.SettableConfig;
+import com.netflix.archaius.config.DefaultCompositeConfig;
+import com.netflix.archaius.config.DefaultSettableConfig;
+
+/**
+ * Implementation of {@link DefaultCompositeConfig} and {@link SettableConfig}
+ * for use in testing utilities. 
+ */
+public class TestCompositeConfig extends DefaultCompositeConfig implements SettableConfig {
+
+    private static final String CLASS_LEVEL_LAYER_NAME =    "CLASS_LEVEL_TEST_OVERRIDES";
+    private static final String METHOD_LEVEL_LAYER_NAME =   "METHOD_LEVEL_TEST_OVERRIDES";
+    private static final String RUNTIME_LAYER_NAME =        "RUNTIME";
+    
+    public TestCompositeConfig(SettableConfig classLevelOverrides, SettableConfig methodLevelOverrides) {
+        try {
+            addConfig(RUNTIME_LAYER_NAME, new DefaultSettableConfig());
```

Comment:
Is it worth holding on to a direct reference to this configuration to avoid the down casts to SettableConfig later in the class?  Alternatively, consider adding a private helper method, getSettableConfig, that localizes the downcast to that method.